.github/workflows/auto-fix-lint.yml

```
@@ -14,19 +14,10 @@ jobs:
     if: github.actor != 'dependabot[bot]' && !github.event.pull_request.head.repo.fork
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: sap/ai-sdk-js/.github/actions/setup@scripts
```

Those all need to be changed to 
```suggestion
      - uses: sap/ai-sdk-js/.github/actions/setup@main
```

before merging!